ludwig/schema/trainer.py

```diff
@@ -64,7 +64,11 @@ class BaseTrainerConfig(schema_utils.BaseMarshmallowConfig):
         default=None,
         allow_none=True,
         min_exclusive=0,
-        description="Size of batch to pass to the model for evaluation.",
+        description=(
```

Comment:
We should also mention `auto`, which will set the eval batch size to maximize GPU bandwidth.

Reply:
Done.